scss/mixins/_grid.scss

```diff
@@ -2,9 +2,9 @@
 //
 // Generate semantic grid columns with these mixins.
 
-@mixin make-row($gutter: $grid-gutter-width) {
-  --#{$variable-prefix}gutter-x: #{$gutter};
-  --#{$variable-prefix}gutter-y: 0;
+@mixin make-row($gutter-x: $grid-gutter-width, $gutter-y: 0) {
```

Comment:
`0` remains hardcoded here. I think the initial issue requested a global Sass variable to handle this, don't you think?

Something like `$grid-gutter-vertical` or something could do the job, with a `!default` value of `0`.

Reply:
You're right, I thought it was already nice to be able to override the vertical gutter with the mixin (less hardcoded), but for some users it will be easier with a variable.

For the wording I chose `height`, for consistency with the existing gutter variable `width`. Maybe it's not perfect, tell me if you prefer another wording. I think we should keep consistency between these 2 variables.